loggly/adapter/adapter.go

```diff
@@ -0,0 +1,139 @@
+package adapter
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/gliderlabs/logspout/router"
+)
+
+const (
+	logglyAddr          = "https://logs-01.loggly.com"
+	logglyEventEndpoint = "/bulk"
+	logglyTagsHeader    = "X-LOGGLY-TAG"
+	flushTimeout        = 10 * time.Second
+)
+
+// Adapter satisfies the router.LogAdapter interface by providing Stream which
+// passes all messages to loggly.
+type Adapter struct {
+	token      string
+	client     *http.Client
+	tags       string
+	log        *log.Logger
+	queue      chan logglyMessage
+	m          sync.Mutex
```

Comment:
Looks like it was meant to protect the http client? According to the source https://golang.org/src/net/http/client.go http.Client uses a RoundTripper interface and all RoundTrippers must be safe to use by multiple go routines by default.